### ISSUE TITLE
Import-AzureVM not working

### ISSUE THREAD

**Issue by cmerat** [Azure/azure-powershell#1945]
Windows 10 Pro. Using SDK version 1.2.3 installed via the Web Platform Installer (shows as 1.2.2 in the WPI).

```
ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   1.2.3      Azure                               {Add-AzureAccount, Add-AzureApplicationGatewaySslCertifica...
```

Getting the following error when attempting to Import-AzureVM

```
PS C:\WINDOWS\system32> Import-AzureVM -Path c:\temp\VM.xml
Import-AzureVM : There was an error reflecting type
'Microsoft.WindowsAzure.Commands.ServiceManagement.Model.PersistentVM'.
At line:1 char:1
+ Import-AzureVM -Path c:\temp\Azure\game2013-Raw.xml
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Import-AzureVM], InvalidOperationException
    + FullyQualifiedErrorId : Microsoft.WindowsAzure.Commands.ServiceManagement.IaaS.ImportAzureVMCommand
```

Using an alternate computer with Windows 8.1 and the 0.9.3 SDK works as expected.

**Comment by markcowl** [comment]
@singhkay Can you take a look?

**Comment by singhkay** [comment]
Probably the same error as #1944. I'll file an investgation work item on the dev team.

**Comment by BorisB2015** [comment]
Put full exception stack in [#1944](https://github.com/Azure/azure-powershell/issues/1944)

**Comment by singhkay** [comment]
This was fixed in #2045. Please give it a try and let us know.
